js/adapt-contrib-media.js

```diff
@@ -39,34 +37,40 @@ define([
    */
   _.extend(window.mejs.MepDefaults, {
     keyActions: [],
-    defaultSeekForwardInterval: function(duration) {
+    defaultSeekForwardInterval: (duration) => {
       if (typeof duration === 'object') return duration.duration * 0.05;
       return duration * 0.05;
     },
-    defaultSeekBackwardInterval: function(duration) {
+    defaultSeekBackwardInterval: (duration) => {
```

Comment:
```suggestion
    defaultSeekBackwardInterval: duration => {
```
Brackets are unnecessary. 

Reference: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions#comparing_traditional_functions_to_arrow_functions